refactor: Removing aria attributes from Banner component (#2318)
Workday/canvas-kit: modules/react/banner/stories/Banner.stories.mdx

<ExampleCodeBlock code={Error} />

### Icon Banner

> When not using any text in the `Banner`, use our `Tooltip` component to both show a visible text alternative, and assign an `aria-label` string to the child `Banner`. 

Reviewer: ```suggestion
When only using an icon in the `Banner`, use our `Tooltip` component to both show a visible text alternative, and assign an `aria-label` string to the child `Banner`. 
```
